### ISSUE TITLE
workload/schemachange: handle crdb_internal computed cols in colIsRefByComputed

### ISSUE THREAD

**Issue by annrpom** [cockroachdb/cockroach#126967]
Our query for colIsRefByComputed cannot handle the following case (human-readable ops generated by schemachange workload):

```
demo@127.0.0.1:26257/demoapp/movr> CREATE TABLE public.table_w0_1 (                                        
                                ->     "co\\u58A1l1 _w0_8" INTERVAL NOT NULL,                              
                                ->     INDEX (                                                             
                                ->         (CASE                                                           
                                ->             WHEN "co\\u58A1l1 _w0_8" IS NULL THEN ')':::STRING          
                                ->             ELSE e'-s\\x12M\\bps':::STRING                              
                                ->         END) DESC                                                       
                                ->     )                                                                   
                                -> );                                                                      
CREATE TABLE

Time: 10ms total (execution 10ms / network 0ms)

demo@127.0.0.1:26257/demoapp/movr> ALTER TABLE public.table_w0_1                                           
                                -> DROP COLUMN "co\\u58A1l1 _w0_8";                                        
ERROR: column "co\\\\u58A1l1 _w0_8" is referenced by computed column "crdb_internal_idx_expr"
SQLSTATE: 42P10
```

Jira issue: CRDB-40208

Epic CRDB-19168

**Comment by annrpom** [comment]
From Faizan: 

> I think this would be similar to: getUniqueConstraintsForTable and the whole unique constraint logic that detects column references.

**Comment by rafiss** [comment]
Let's fix this as part of enabling DROP COLUMN (#127286): https://github.com/cockroachdb/cockroach/blob/e77f7145617fe9a14bb3e50b032f6804d4459183/pkg/workload/schemachange/operation_generator.go#L1631-L1635
